src/jaxsim/simulation/integrators.py

```diff
@@ -230,6 +170,7 @@ def body_fun(carry: Carry, xs: None) -> Tuple[Carry, None]:
         # -----------------------------------------------------
 
         # Extract the implicit angular velocity and the initial base quaternion
+
```

Comment:
remove empty line?

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/82/commits/29a6ba3e9eb3aa0f7597897aaa5eb843a1bf745e